File: MODULE 3/3A_JAVASCRIPTS/JAVASCRIPTS CODING CHALLENGE/Task10.js
// JAVASCRIPT CODING CHALLENGE

// 10. Create a function that filters out negative numbers.

function filternegatives(numbers) {
    //const positiveNumbers = numbers.filter(num => num >= 0);
    const negativeNumbers = numbers.filter(num => num < 0);
    return negativeNumbers;
  }
  
  const numbers = [-5, 0, 7, -2, 10];
  const filteredNumbers = filternegatives(numbers);
  console.log(filteredNumbers); // Output: [0, 7, 10]

File: MODULE 4C/PYTHON_CODING_CHALLENGE/Task10.py
"""
PYTHON CODING CHALLENGE
10. Create a function that filters out negative numbers.
"""
def filter_negative_numbers(arr):
    non_negative_numbers = []
    for num in arr:
        if num >= 0:
            non_negative_numbers.append(num)
    print("Filtered Non-Negative Numbers:", non_negative_numbers)
    return non_negative_numbers
